src/operator/random/sample_op.h

```diff
@@ -78,6 +78,11 @@ struct GenNegBinomialParam {
   float alpha;
 };
 
+struct RandIntParam {
+  int low;
```

Comment:
What if low is > 4B and dtype is int64? Overflow bug?

Reply:
converting that to int64_t